dockerHashing/src/test/scala/cromwell/docker/registryv2/AlibabaCloudCRRegistrySpec.scala

```diff
@@ -19,7 +19,7 @@ object AlibabaCloudCRRegistrySpec {
        |enable = true
        |# How should docker hashes be looked up. Possible values are "local" and "remote"
        |# "local": Lookup hashes on the local docker daemon using the cli
-       |# "remote": Lookup hashes on docker hub and gcr
+       |# "remote": Lookup hashes on docker hub, gcr, and gar
```

Comment:
ToL: Of all the duplicates in the codebase of this string (maybe 4 at my rough count [[0]](https://github.com/broadinstitute/cromwell/blob/65/dockerHashing/src/test/scala/cromwell/docker/registryv2/AlibabaCloudCRRegistrySpec.scala#L22) [[1]](https://github.com/broadinstitute/cromwell/blob/65/cromwell.example.backends/cromwell.examples.conf#L241) [[2]](https://github.com/broadinstitute/cromwell/blob/65/core/src/main/resources/reference.conf#L364) [[3]](https://github.com/broadinstitute/cromwell/blame/65/docs/tutorials/Containers.md#L460))... why was this copy updated, and only this copy?

This instance of the copy-pasted `method` key isn't even used and should probably be deleted at some point. The code below that is only looking for the `alibabacloudcr` stanza.